pkg/vz/network_darwin.go

```diff
@@ -27,7 +28,7 @@ func PassFDToUnix(unixSock string) (*os.File, error) {
 	if err != nil {
 		return nil, err
 	}
-	err = fd.Put(unixConn.(*net.UnixConn), server)
+	err = fd.Put(unixConn, server)
```

Comment:
Here we get rid of the type assertion, so making the code little worse has somme benefit.